qiita_pet/templates/list_studies.html

```diff
@@ -0,0 +1,266 @@
+{% extends sitebase.html %}
+{% block head %}
+<link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
+
+<script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
+
+<script type="text/javascript">
+var current_study;
+var ajaxURL = "/study/search/?&user={{current_user.id}}&sEcho=" + Math.floor(Math.random()*1001);
+$(document).ready(function() {
+        $('#standard-studies-table').dataTable({
+            "columns": [
+              { "data": "checkbox" },
+              { "data": "title" },
+              { "data": "abstract" },
+              { "data": "id" },
+              { "data": "meta_complete" },
+              { "data": "num_samples" },
+              { "data": "num_raw_data" },
+              { "data": "shared" },
+              { "data": "pi" },
+              { "data": "pmid" },
+              { "data": "status" }
+            ],
+            order: [[10, "asc"], [ 1, "asc" ]],
```

Comment:
Does this sort first by status and then by title? I would think sorting just by title would be a user's expectation, but I am not sure